package.json

```diff
@@ -11,7 +11,7 @@
     "linkAll": "lerna exec 'npm link' --parallel",
     "unlinkAll": "lerna exec 'npm unlink' --parallel",
     "dev": "npm run clean && npm run watch",
-    "prettier": "prettier \"packages/*/{src/**,examples/**/src/**}.{md,js,jsx,ts,tsx,json}\" --write",
+    "prettier": "prettier \"{packages/*/src/**,examples/*/*/src/**}.{md,js,jsx,ts,tsx,vue,json}\" --write",
```

Comment:
Added the `.vue` extension and fixed the `prettier` command `glob` so that it takes into account the examples as well.